fixtures/features/api-elements/param-with-enum-default.json

```diff
@@ -86,7 +86,9 @@
                           "meta": {},
                           "attributes": {
                             "default": [
-                              "A"
```

Comment:
Not sure, what happen there. It is "regenerated test"
Will investigate why it happen.